### PR TITLE
Create an Elastic Defend policy through API

### LINKED ISSUE #3329: Document how to create an Elastic Defend policy through API
## Description 
We've had some customer confusion on how to create an Elastic Defend policy through the API.  We should document the steps to take specific to Elastic Defend.

It will follow the steps from this discuss issue: https://discuss.elastic.co/t/defend-api-integration/329487

## Acceptance Test Criteria

- As a user, I should use be able to us the Policy API to create Elastic Defend policies
 

## Notes

- Add all appropriate labels to the issue, especially the version number label. 
- Be sure to add any necessary screenshots, code text or console commands for clarity. 
- Include any conditions or caveats that may affect customers.

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves #3329.

Previews:
- [Create an Elastic Defend policy using API](https://security-docs_3661.docs-preview.app.elstc.co/guide/en/security/master/create-defend-policy-api.html) – new page
- [Configure an integration policy for Elastic Defend](https://security-docs_3661.docs-preview.app.elstc.co/guide/en/security/master/configure-endpoint-integration-policy.html) – tip callout added after **Requirements**

### CODE REVIEW

**Comment by natasha-moore-elastic** [comment]
@kevinlog, are there any requirements in terms of privileges or similar that users must have to use this API?
Is [this requirement](https://www.elastic.co/guide/en/security/current/configure-endpoint-integration-policy.html) relevant here?

> You must have the **Elastic Defend Policy Management : All** [privilege](https://www.elastic.co/guide/en/security/current/endpoint-management-req.html) to configure an integration policy.

File: docs/getting-started/create-defend-policy-api.asciidoc [hunk 1]
```diff
@@ -0,0 +1,813 @@
...
+:frontmatter-tags-content-type: [how-to]
+:frontmatter-tags-user-goals: [manage]
+
+In addition to <<configure-endpoint-integration-policy, configuring an {elastic-defend} policy>> through the {elastic-sec} UI, you can create and customize an {elastic-defend} policy through the API. This is a three-step process involving the {fleet} API. For detailed information on {fleet} API, refer to {fleet-guide}/fleet-api-docs.html[{fleet} API documentation].
+
+[discrete]
+[[create-agent-policy]]
+== Step 1: Create an agent policy
+
+Make a call to create an agent policy that you will add your {elastic-defend} integration to. Replace `<KIBANA-VERSION>` with your version of {kib}.
```
Comment (benironside) [inline docs/getting-started/create-defend-policy-api.asciidoc:15]:
```suggestion
Use the following API call to create a new agent policy where you will add your {elastic-defend} integration. Replace `<KIBANA-VERSION>` with your version of {kib}.
```
Maybe worth specifying "API call" the first few times "call" appears in this doc

File: docs/getting-started/create-defend-policy-api.asciidoc [hunk 2]
```diff
@@ -0,0 +1,813 @@
...
+[[add-defend-integration]]
+== Step 2: Add the {elastic-defend} integration
+
+Next, add the {elastic-defend} integration to the policy that you created in step 1.
+
+In the following call:
+
+. Replace `<KIBANA-VERSION>` with your version of {kib}.
+. Replace `<POLICY-ID>` with the agent policy ID you received in step 1.
+. Replace `<LATEST-ELASTIC-DEFEND-PACKAGE-VERSION>` with the latest {elastic-defend} package version. To find this, go to **Management** -> **Integrations** and select *{elastic-defend}*. This will be a version such as 8.9.1.
```
Comment (benironside) [inline docs/getting-started/create-defend-policy-api.asciidoc:83]:
```suggestion
. Replace `<LATEST-ELASTIC-DEFEND-PACKAGE-VERSION>` with the latest {elastic-defend} package version (for example, `8.9.1`). To find it, go to **Management** -> **Integrations** and select *{elastic-defend}*. 
```

File: docs/getting-started/create-defend-policy-api.asciidoc [hunk 3]
```diff
@@ -0,0 +1,813 @@
...
+    "version": "<LATEST-ELASTIC-DEFEND-PACKAGE-VERSION>" <3>
+  }
+}' \
+  --compressed
+----
+<1> `<KIBANA-VERSION>` to be replaced
+<2> `<POLICY-ID>` to be replaced
+<3> `<LATEST-ELASTIC-DEFEND-PACKAGE-VERSION>` to be replaced
+
+Make a note of the `<PACKAGE-POLICY-ID>` you receive in the response. This refers to the {elastic-defend} policy and you will use this when making a call in step 3.
```
Comment (benironside) [inline docs/getting-started/create-defend-policy-api.asciidoc:134]:
```suggestion
Make a note of the `<PACKAGE-POLICY-ID>` you receive in the response. This refers to the {elastic-defend} policy and you will use it in step 3.
```

File: docs/getting-started/create-defend-policy-api.asciidoc [hunk 4]
```diff
@@ -0,0 +1,813 @@
...
+"id": "<PACKAGE-POLICY-ID>",
+"revision": 1,
+"created_at": "2023-04-06T15:53:14.020Z",
+"created_by": "elastic",
+"updated_at": "2023-04-06T15:53:14.020Z",
+"updated_by": "elastic"
+----
+. Make any changes to the `policy` object to customize the {elastic-defend} configuration.
+
+Using the resulting data, save the modified {elastic-defend} policy.
```
Comment (benironside) [inline docs/getting-started/create-defend-policy-api.asciidoc:478]:
```suggestion
Include the resulting JSON object in the following call to modify the {elastic-defend} policy. Replace these fields:
```
Maybe my suggestion is too wordy, but I found this sentence pretty confusing in isolation because it didn't specify how to do the thing it was telling me how to do. I had to skip several lines down to figure out how to do what it was telling me

File: docs/getting-started/create-defend-policy-api.asciidoc [hunk 5]
```diff
@@ -0,0 +1,813 @@
...
+
+In the following call:
+
+. Replace `<PACKAGE-POLICY-ID>` with the {elastic-defend} policy ID you received in step 2.
+. Replace `<KIBANA-VERSION>` with your version of {kib}.
+. Replace `<LATEST-ELASTIC-DEFEND-PACKAGE-VERSION>` with the latest {elastic-defend} package version. To find this, go to **Management** -> **Integrations** and select *{elastic-defend}*. This will be a version such as 8.9.1.
```
Comment (benironside) [inline docs/getting-started/create-defend-policy-api.asciidoc:484]:
```suggestion
. Replace `<LATEST-ELASTIC-DEFEND-PACKAGE-VERSION>` with the latest {elastic-defend} package version (for example, `8.9.1`). To find it, go to **Management** -> **Integrations** and select *{elastic-defend}*. 
```

File: docs/getting-started/create-defend-policy-api.asciidoc [hunk 6]
```diff
@@ -0,0 +1,813 @@
...
+    }
+  ]
+}' \
+  --compressed
+----
+<1> `<PACKAGE-POLICY-ID>` to be replaced
+<2> `<KIBANA-VERSION>` to be replaced
+<3> `<LATEST-ELASTIC-DEFEND-PACKAGE-VERSION>` to be replaced
+
+You can repeat steps 2 and 3 if you need to make more modifications to the {elastic-defend} policy.
```
Comment (benironside) [inline docs/getting-started/create-defend-policy-api.asciidoc:813]:
```suggestion
You can repeat steps 2 and 3 to make more modifications to the {elastic-defend} policy.
```

File: docs/getting-started/create-defend-policy-api.asciidoc [hunk 7]
```diff
@@ -0,0 +1,813 @@
...
+  }
+}
+----
+<1> `<PACKAGE-POLICY-ID>` needed in step 3
+
+[discrete]
+[[customize-policy-settings]]
+== Step 3: Customize the {elastic-defend} policy settings
+
+With the response received in step 2, do the following:
```
Comment (benironside) [inline docs/getting-started/create-defend-policy-api.asciidoc:462]:
```suggestion
The response received in step 2 represents the configuration of your new integration. To modify its configuration, you'll make another API call with a similar structure but different values:
```

**Review by benironside — COMMENTED** [review]
Great work, I really like that this provides a walkthrough for using the API. Left some suggestions, I hope they're helpful, let me know if you have any questions. One bigger question is whether it might be appropriate to make the code blocks collapsible — I had to do a lot of scrolling while figuring out the new page and it would have saved me time.

File: docs/getting-started/create-defend-policy-api.asciidoc [hunk 8]
```diff
@@ -0,0 +1,813 @@
+[[create-defend-policy-api]]
+= Create an {elastic-defend} policy using API
+
+:frontmatter-description: Create and customize an {elastic-defend} policy through the API.
+:frontmatter-tags-products: [security, defend]
+:frontmatter-tags-content-type: [how-to]
+:frontmatter-tags-user-goals: [manage]
+
+In addition to <<configure-endpoint-integration-policy, configuring an {elastic-defend} policy>> through the {elastic-sec} UI, you can create and customize an {elastic-defend} policy through the API. This is a three-step process involving the {fleet} API. For detailed information on {fleet} API, refer to {fleet-guide}/fleet-api-docs.html[{fleet} API documentation].
```
[the remark this answers is not in the document]
Comment (joepeeples) [reply docs/getting-started/create-defend-policy-api.asciidoc:9]: Shorter alternative — just include the link, no need to explain what it is. Most readers know that links point to more detailed info or background on the topic being linked:
```suggestion
In addition to <<configure-endpoint-integration-policy, configuring an {elastic-defend} policy>> through the {elastic-sec} UI, you can create and customize an {elastic-defend} policy through the API. This is a three-step process involving the {fleet-guide}/fleet-api-docs.html[{fleet} API].
```

File: docs/getting-started/create-defend-policy-api.asciidoc (hunk 4 above)
Comment (joepeeples) [reply docs/getting-started/create-defend-policy-api.asciidoc:478]: I can see how @benironside's edit helps clarify things. In addition, this line feels like it's another step in the numbered procedure above, but it's formatted as plain text. And then there's another stepped procedure below this -- maybe combine all into a continuous stepped procedure?

File: docs/getting-started/create-defend-policy-api.asciidoc (hunk 6 above)
Comment (joepeeples) [reply docs/getting-started/create-defend-policy-api.asciidoc:813]: This line feels buried under the lengthy code example, and I feel like it could get overlooked. WDYT about moving it all the way to the top of the page, in the intro, to let readers know from the beginning how they can further modify the policy? Something like:

"This is a three-step process involving the Fleet API. You can repeat steps 2 and 3 to make more modifications..."

**Review by joepeeples — COMMENTED** [review]
Great work, thank you for preparing these docs! Added a few additional suggestions.

**Comment by natasha-moore-elastic** [comment]
Thanks for the great feedback, @benironside & @joepeeples! I've made changes to clarify some of the steps per your suggestions.

**Comment by natasha-moore-elastic** [comment]
> Great work, I really like that this provides a walkthrough for using the API. Left some suggestions, I hope they're helpful, let me know if you have any questions. One bigger question is whether it might be appropriate to make the code blocks collapsible — I had to do a lot of scrolling while figuring out the new page and it would have saved me time.

Good point and something that Joe and I discussed, too. Originally, I didn't make them collapsible because I felt it would hide relevant information re the values that users need to take note of. But since I've rearranged that content, I've now made the response examples collapsible since the info around the key values now precedes them. I'm not in favor of collapsing the request examples though, since those are an integral part of the walkthrough.

File: docs/getting-started/create-defend-policy-api.asciidoc (hunk 4 above)
Comment (natasha-moore-elastic) [reply docs/getting-started/create-defend-policy-api.asciidoc]: Agree this could have been more clear. I've used subsections to rearrange the content, which should hopefully make it clearer that step 3 consists of two sub-steps (1. modify the default policy configuration and 2. save the customized configuration).

File: docs/getting-started/create-defend-policy-api.asciidoc [hunk 9]
```diff
@@ -0,0 +1,819 @@
...
+  "description": "",
+  "namespace": "default",
+  "monitoring_enabled": [
+    "logs",
+    "metrics"
+  ],
+  "inactivity_timeout": 1209600,
+  "is_protected": false
+}' \
+  --compressed
```
Comment (kevinlog) [inline docs/getting-started/create-defend-policy-api.asciidoc:40]: Can we actually write this like below?  This will make the curl more copy/paste-able for users.  Apologies, I should have listed these calls like this before.

```
curl --user <username>:<password> --request POST \
  --url 'https://<kibana-url>:5601/api/fleet/agent_policies' \
  -H 'Accept: */*' \
  -H 'Accept-Language: en-US,en;q=0.9' \
  -H 'Connection: keep-alive' \
  -H 'Content-Type: application/json' \
  -H 'Sec-Fetch-Dest: empty' \
  -H 'Sec-Fetch-Mode: cors' \
  -H 'Sec-Fetch-Site: same-origin' \
  -H 'kbn-version: <KIBANA-VERSION>' \ <1>
  -d \
'
{
  "name": "My Policy Name",
  "description": "",
  "namespace": "default",
  "monitoring_enabled": [
    "logs",
    "metrics"
  ],
  "inactivity_timeout": 1209600,
  "is_protected": false
}'
```

File: docs/getting-started/create-defend-policy-api.asciidoc [hunk 10]
```diff
@@ -0,0 +1,819 @@
...
+                "file": "info"
+              }
+            }
+          }
+        }
+      }
+    }
+  ]
+}' \
+  --compressed
```
Comment (kevinlog) [inline docs/getting-started/create-defend-policy-api.asciidoc:815]: Similarly for this, can we write it like the below?

```
curl --user <username>:<password> --request PUT \
  --url 'https://<kibana-url>:5601/api/fleet/package_policies/<PACKAGE-POLICY-ID>' \ <1>
  -H 'Accept: */*' \
  -H 'Accept-Language: en-US,en;q=0.9' \
  -H 'Connection: keep-alive' \
  -H 'Content-Type: application/json' \
  -H 'Sec-Fetch-Dest: empty' \
  -H 'Sec-Fetch-Mode: cors' \
  -H 'Sec-Fetch-Site: same-origin' \
  -H 'kbn-version: <KIBANA-VERSION>' \ <2>
  -d \
'
{
  "version": "WzMwOTcsMV0=",
  "name": "Protect",
  "namespace": "default",
  "description": "",
  "package": {
    "name": "endpoint",
    "title": "Elastic Defend",
    "version": "<LATEST-ELASTIC-DEFEND-PACKAGE-VERSION>" <3> 
  },
  "enabled": true,
  "policy_id": "b4be0860-d492-11ed-a59c-3ffbbd16325a",
  "inputs": [
    {
      "type": "endpoint",
      "enabled": true,
      "streams": [],
      "config": {
        "integration_config": {
          "value": {
            "type": "endpoint",
            "endpointConfig": {
              "preset": "EDRComplete"
            }
          }
        },
        "artifact_manifest": {
          "value": {
            "manifest_version": "1.0.2",
            "schema_version": "v1",
            "artifacts": {
              "endpoint-exceptionlist-macos-v1": {
                "encryption_algorithm": "none",
                "decoded_sha256": "d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "decoded_size": 14,
                "encoded_sha256": "f8e6afa1d5662f5b37f83337af774b5785b5b7f1daee08b7b00c2d6813874cda",
                "encoded_size": 22,
                "relative_url": "/api/fleet/artifacts/endpoint-exceptionlist-macos-v1/d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "compression_algorithm": "zlib"
              },
              "endpoint-exceptionlist-windows-v1": {
                "encryption_algorithm": "none",
                "decoded_sha256": "d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "decoded_size": 14,
                "encoded_sha256": "f8e6afa1d5662f5b37f83337af774b5785b5b7f1daee08b7b00c2d6813874cda",
                "encoded_size": 22,
                "relative_url": "/api/fleet/artifacts/endpoint-exceptionlist-windows-v1/d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "compression_algorithm": "zlib"
              },
              "endpoint-exceptionlist-linux-v1": {
                "encryption_algorithm": "none",
                "decoded_sha256": "d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "decoded_size": 14,
                "encoded_sha256": "f8e6afa1d5662f5b37f83337af774b5785b5b7f1daee08b7b00c2d6813874cda",
                "encoded_size": 22,
                "relative_url": "/api/fleet/artifacts/endpoint-exceptionlist-linux-v1/d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "compression_algorithm": "zlib"
              },
              "endpoint-trustlist-macos-v1": {
                "encryption_algorithm": "none",
                "decoded_sha256": "d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "decoded_size": 14,
                "encoded_sha256": "f8e6afa1d5662f5b37f83337af774b5785b5b7f1daee08b7b00c2d6813874cda",
                "encoded_size": 22,
                "relative_url": "/api/fleet/artifacts/endpoint-trustlist-macos-v1/d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "compression_algorithm": "zlib"
              },
              "endpoint-trustlist-windows-v1": {
                "encryption_algorithm": "none",
                "decoded_sha256": "d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "decoded_size": 14,
                "encoded_sha256": "f8e6afa1d5662f5b37f83337af774b5785b5b7f1daee08b7b00c2d6813874cda",
                "encoded_size": 22,
                "relative_url": "/api/fleet/artifacts/endpoint-trustlist-windows-v1/d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "compression_algorithm": "zlib"
              },
              "endpoint-trustlist-linux-v1": {
                "encryption_algorithm": "none",
                "decoded_sha256": "d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "decoded_size": 14,
                "encoded_sha256": "f8e6afa1d5662f5b37f83337af774b5785b5b7f1daee08b7b00c2d6813874cda",
                "encoded_size": 22,
                "relative_url": "/api/fleet/artifacts/endpoint-trustlist-linux-v1/d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "compression_algorithm": "zlib"
              },
              "endpoint-eventfilterlist-macos-v1": {
                "encryption_algorithm": "none",
                "decoded_sha256": "d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "decoded_size": 14,
                "encoded_sha256": "f8e6afa1d5662f5b37f83337af774b5785b5b7f1daee08b7b00c2d6813874cda",
                "encoded_size": 22,
                "relative_url": "/api/fleet/artifacts/endpoint-eventfilterlist-macos-v1/d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "compression_algorithm": "zlib"
              },
              "endpoint-eventfilterlist-windows-v1": {
                "encryption_algorithm": "none",
                "decoded_sha256": "d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "decoded_size": 14,
                "encoded_sha256": "f8e6afa1d5662f5b37f83337af774b5785b5b7f1daee08b7b00c2d6813874cda",
                "encoded_size": 22,
                "relative_url": "/api/fleet/artifacts/endpoint-eventfilterlist-windows-v1/d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "compression_algorithm": "zlib"
              },
              "endpoint-eventfilterlist-linux-v1": {
                "encryption_algorithm": "none",
                "decoded_sha256": "d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "decoded_size": 14,
                "encoded_sha256": "f8e6afa1d5662f5b37f83337af774b5785b5b7f1daee08b7b00c2d6813874cda",
                "encoded_size": 22,
                "relative_url": "/api/fleet/artifacts/endpoint-eventfilterlist-linux-v1/d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "compression_algorithm": "zlib"
              },
              "endpoint-hostisolationexceptionlist-macos-v1": {
                "encryption_algorithm": "none",
                "decoded_sha256": "d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "decoded_size": 14,
                "encoded_sha256": "f8e6afa1d5662f5b37f83337af774b5785b5b7f1daee08b7b00c2d6813874cda",
                "encoded_size": 22,
                "relative_url": "/api/fleet/artifacts/endpoint-hostisolationexceptionlist-macos-v1/d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "compression_algorithm": "zlib"
              },
              "endpoint-hostisolationexceptionlist-windows-v1": {
                "encryption_algorithm": "none",
                "decoded_sha256": "d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "decoded_size": 14,
                "encoded_sha256": "f8e6afa1d5662f5b37f83337af774b5785b5b7f1daee08b7b00c2d6813874cda",
                "encoded_size": 22,
                "relative_url": "/api/fleet/artifacts/endpoint-hostisolationexceptionlist-windows-v1/d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "compression_algorithm": "zlib"
              },
              "endpoint-hostisolationexceptionlist-linux-v1": {
                "encryption_algorithm": "none",
                "decoded_sha256": "d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "decoded_size": 14,
                "encoded_sha256": "f8e6afa1d5662f5b37f83337af774b5785b5b7f1daee08b7b00c2d6813874cda",
                "encoded_size": 22,
                "relative_url": "/api/fleet/artifacts/endpoint-hostisolationexceptionlist-linux-v1/d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "compression_algorithm": "zlib"
              },
              "endpoint-blocklist-macos-v1": {
                "encryption_algorithm": "none",
                "decoded_sha256": "d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "decoded_size": 14,
                "encoded_sha256": "f8e6afa1d5662f5b37f83337af774b5785b5b7f1daee08b7b00c2d6813874cda",
                "encoded_size": 22,
                "relative_url": "/api/fleet/artifacts/endpoint-blocklist-macos-v1/d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "compression_algorithm": "zlib"
              },
              "endpoint-blocklist-windows-v1": {
                "encryption_algorithm": "none",
                "decoded_sha256": "d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "decoded_size": 14,
                "encoded_sha256": "f8e6afa1d5662f5b37f83337af774b5785b5b7f1daee08b7b00c2d6813874cda",
                "encoded_size": 22,
                "relative_url": "/api/fleet/artifacts/endpoint-blocklist-windows-v1/d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "compression_algorithm": "zlib"
              },
              "endpoint-blocklist-linux-v1": {
                "encryption_algorithm": "none",
                "decoded_sha256": "d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "decoded_size": 14,
                "encoded_sha256": "f8e6afa1d5662f5b37f83337af774b5785b5b7f1daee08b7b00c2d6813874cda",
                "encoded_size": 22,
                "relative_url": "/api/fleet/artifacts/endpoint-blocklist-linux-v1/d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658",
                "compression_algorithm": "zlib"
              }
            }
          }
        },
        "policy": {
          "value": {
            "windows": {
              "events": {
                "dll_and_driver_load": true,
                "dns": true,
                "file": true,
                "network": true,
                "process": true,
                "registry": true,
                "security": true
              },
              "malware": {
                "mode": "prevent",
                "blocklist": true
              },
              "ransomware": {
                "mode": "prevent",
                "supported": true
              },
              "memory_protection": {
                "mode": "prevent",
                "supported": true
              },
              "behavior_protection": {
                "mode": "prevent",
                "supported": true
              },
              "popup": {
                "malware": {
                  "message": "",
                  "enabled": true
                },
                "ransomware": {
                  "message": "",
                  "enabled": true
                },
                "memory_protection": {
                  "message": "",
                  "enabled": true
                },
                "behavior_protection": {
                  "message": "",
                  "enabled": true
                }
              },
              "logging": {
                "file": "info"
              },
              "antivirus_registration": {
                "enabled": false
              },
              "attack_surface_reduction": {
                "credential_hardening": {
                  "enabled": true
                }
              }
            },
            "mac": {
              "events": {
                "process": true,
                "file": true,
                "network": true
              },
              "malware": {
                "mode": "prevent",
                "blocklist": true
              },
              "behavior_protection": {
                "mode": "prevent",
                "supported": true
              },
              "memory_protection": {
                "mode": "prevent",
                "supported": true
              },
              "popup": {
                "malware": {
                  "message": "",
                  "enabled": true
                },
                "behavior_protection": {
                  "message": "",
                  "enabled": true
                },
                "memory_protection": {
                  "message": "",
                  "enabled": true
                }
              },
              "logging": {
                "file": "info"
              }
            },
            "linux": {
              "events": {
                "process": true,
                "file": true,
                "network": true,
                "session_data": false,
                "tty_io": false
              },
              "malware": {
                "mode": "prevent",
                "blocklist": true
              },
              "behavior_protection": {
                "mode": "prevent",
                "supported": true
              },
              "memory_protection": {
                "mode": "prevent",
                "supported": true
              },
              "popup": {
                "malware": {
                  "message": "",
                  "enabled": true
                },
                "behavior_protection": {
                  "message": "",
                  "enabled": true
                },
                "memory_protection": {
                  "message": "",
                  "enabled": true
                }
              },
              "logging": {
                "file": "info"
              }
            }
          }
        }
      }
    }
  ]
}'
```

File: docs/getting-started/create-defend-policy-api.asciidoc [hunk 11]
```diff
@@ -0,0 +1,819 @@
...
+      }
+    }
+  ],
+  "package": {
+    "name": "endpoint",
+    "title": "Elastic Defend",
+    "version": "<LATEST-ELASTIC-DEFEND-PACKAGE-VERSION>" <3>
+  }
+}' \
+  --compressed
```
Comment (kevinlog) [inline docs/getting-started/create-defend-policy-api.asciidoc:130]: And for this one, can we write it like below?

```
curl --user <username>:<password> --request POST \
  --url 'https://<kibana-url>:5601/api/fleet/package_policies' \
  -H 'Accept: */*' \
  -H 'Accept-Language: en-US,en;q=0.9' \
  -H 'Connection: keep-alive' \
  -H 'Content-Type: application/json' \
  -H 'Sec-Fetch-Dest: empty' \
  -H 'Sec-Fetch-Mode: cors' \
  -H 'Sec-Fetch-Site: same-origin' \
  -H 'kbn-version: <KIBANA-VERSION>' \ <1>
  -d \
'
{
  "name": "Protect",
  "description": "",
  "namespace": "default",
  "policy_id": "<POLICY-ID>", <2>
  "enabled": true,
  "inputs": [
    {
      "enabled": true,
      "streams": [],
      "type": "ENDPOINT_INTEGRATION_CONFIG",
      "config": {
        "_config": {
          "value": {
            "type": "endpoint",
            "endpointConfig": {
              "preset": "EDRComplete"
            }
          }
        }
      }
    }
  ],
  "package": {
    "name": "endpoint",
    "title": "Elastic Defend",
    "version": "<LATEST-ELASTIC-DEFEND-PACKAGE-VERSION>" <3>
  }
}' 
```

File: docs/getting-started/create-defend-policy-api.asciidoc [hunk 12]
```diff
@@ -0,0 +1,819 @@
...
+====
+[source, json]
+----
+{
+  "item": {
+    "id": "<POLICY-ID>", <1>
+    "name": "My Policy Name",
+    "description": "",
+    "namespace": "default",
+    "monitoring_enabled": [
```
Comment (kevinlog) [inline docs/getting-started/create-defend-policy-api.asciidoc:57]: you can delete `"monitoring_enabled": ["logs", "metrics"]` here to a more consistent looking output.

File: docs/getting-started/create-defend-policy-api.asciidoc [hunk 13]
```diff
@@ -0,0 +1,819 @@
...
+      "logs",
+      "metrics"
+    ],
+    "inactivity_timeout": 1209600,
+    "is_protected": false,
```
Comment (kevinlog) [inline docs/getting-started/create-defend-policy-api.asciidoc:62]: this field you should leave in even though we deleted from the request above.  It's an optional field that will get added whether you include it or not.  This is not an important detail for the purposes of this PR, I just looking to simplify the request where I could.

**Review by kevinlog — APPROVED** [review]
I left a few comments that I think we should address, but giving a preemptive 👍 since I'll be out on Friday.  Thanks for doing this!

**Comment by natasha-moore-elastic** [comment]
> I left a few comments that I think we should address, but giving a preemptive 👍 since I'll be out on Friday. Thanks for doing this!

Thanks @kevinlog! I've pushed up some updates for these suggestions. Just have one final question [here](https://github.com/elastic/security-docs/pull/3661#issuecomment-1663768887) .

File: docs/getting-started/create-defend-policy-api.asciidoc (hunk 13 above)
Comment (natasha-moore-elastic) [reply docs/getting-started/create-defend-policy-api.asciidoc:54]: Got it, thanks for the explanation!

**Comment by kevinlog** [comment]
@natasha-moore-elastic apologies, I missed this comment

https://github.com/elastic/security-docs/pull/3661#issuecomment-1663768887

Yes, users will need these permissions in order to use the API.  Good catch!

File: docs/getting-started/create-defend-policy-api.asciidoc [hunk 14]
```diff
@@ -0,0 +1,819 @@
...
+  "description": "",
+  "namespace": "default",
+  "inactivity_timeout": 1209600
+}'
+----
+<1> `<KIBANA-VERSION>` to be replaced
+
+Make a note of the `<POLICY-ID>` you receive in the response. You will use this in step 2 to add {elastic-defend}.
+
+.Click to see example response
```
Comment (joepeeples) [inline docs/getting-started/create-defend-policy-api.asciidoc:48]: Revise for more more neutral, less ableist language. Could also be something like "Click to expand example response" or just "Click for...":
```suggestion
.Click to display example response
```

File: docs/getting-started/create-defend-policy-api.asciidoc [hunk 15]
```diff
@@ -0,0 +1,819 @@
...
+  }
+}'
+----
+<1> `<KIBANA-VERSION>` to be replaced
+<2> `<POLICY-ID>` to be replaced
+<3> `<LATEST-ELASTIC-DEFEND-PACKAGE-VERSION>` to be replaced
+
+Make a note of the `<PACKAGE-POLICY-ID>` you receive in the response. This refers to the {elastic-defend} policy and you will use it in step 3.
+
+.Click to see example response
```
Comment (joepeeples) [inline docs/getting-started/create-defend-policy-api.asciidoc:137]:
```suggestion
.Click to display example response
```

**Review by joepeeples — COMMENTED** [review]
Looks great! Just one small edit suggestion with accessibility in mind.
